examples/edpm-configmap.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
```

Comment:
I'm wondering if we can just have a single `ConfigMap` (merge basically `edpm-configmap.yaml` and `plugin-configmap.yaml`) and simplify the mounts on the `ansibleee-plugin.yaml` file.
Also, from a user perspective, I think it's better having a single point containing all the required info.

Reply:
So just keeping one configmap for the actual inventory and then writing the plugin-configmap through the `inventory` resource? I like that approach, keeps it super simple. Also the original comments for the direction of this PR might be misaligned to the adjusted scope - i'll ping you tomorrow morning and early next week so we can sync on the operator level items.